jwst/assign_mtwcs/moving_target_wcs.py

```diff
@@ -46,19 +46,29 @@ def assign_moving_target_wcs(input_models):
 
     # loop over only science exposures in the ModelLibrary
     ind = input_models.indices_for_exptype("science")
-    mt_ra = np.empty(len(ind))
-    mt_dec = np.empty(len(ind))
+    mt_ra = np.full(len(ind), None)
+    mt_dec = np.full(len(ind), None)
```

Comment:
I think the regression test diffs are precision related, from this line.  The previous version made an empty float64 array for mt_ra and dec; this one makes an array with object dtype.  Try this and see if it fixes the regtest issues:
```suggestion
    mt_ra = np.full(len(ind), np.nan)
    mt_dec = np.full(len(ind), np.nan)
```

Reply:
Thank you!